packages/block-library/src/table/edit.js

```diff
@@ -498,11 +498,13 @@ export class TableEdit extends Component {
 						] }
 					/>
 				</InspectorControls>
-				<table className={ classes }>
-					<Section type="head" rows={ head } />
-					<Section type="body" rows={ body } />
-					<Section type="foot" rows={ foot } />
-				</table>
+				<div className={ className }>
```

Comment:
How about changing this to `<figure>`, so the markup is clean, we can allow a caption, and it's the same as all our other media/figure blocks? Cc @talldan.

Reply:
Done!